apis/pipeline/pipeline.yaml

```diff
@@ -6535,6 +6537,23 @@ components:
                     additionalProperties:
                         $ref: '#/components/schemas/ResourceSummary'
 
+        NodePoolStatusVSphere:
+            type: object
+            properties:
```

Comment:
isn't template missing from here?

Reply:
you're right, thx, i updated the OpenAPI